package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.8.4",
+  "version": "0.8.5",
```

Comment:
Can you break the version bump into a separate PR?

PRs get squashed into a single commit so we want the ability to easily revert functionality.

Reply:
@marktran Absolutely, I'll update shortly 👍

Reply:
Also, since we're adding functionality we should be bumping the minor version to follow [SemVer](https://semver.org/):
> 2. MINOR version when you add functionality in a backwards compatible manner, and

Reply:
Added a commit reverting package.json version and snapshots back to 0.8.4

Reply:
@maxdeviant I can plan for a bump to 0.9.0 (sorry, bit of a new area for me 😅 )